### DISCUSSION TITLE
第2节课笔记(25班)

### DISCUSSION THREAD

**Discussion by zcg-joker** [InternLM/Tutorial#254] [笔记]
笔记写在这里！
作业**不**写在这里！

**Comment by sxswz213** [comment]
https://blog.csdn.net/sxswz/article/details/135508129?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135508129%22%2C%22source%22%3A%22sxswz%22%7D

**Reply by zcg-joker** [reply]
太酷啦！

**Comment by BaiYu96** [comment]
https://github.com/BaiYu96/internLM-learm/blob/main/note/02-%E4%B9%A6%E7%94%9F%E5%A4%A7%E6%A8%A1%E5%9E%8BDemo%E4%BD%93%E9%AA%8C.md

**Reply by zcg-joker** [reply]
看到你的探索啦~太酷啦！

**Comment by FableFatale** [comment]
https://www.cnblogs.com/yunyetui/p/17957523

**Reply by zcg-joker** [reply]
![image](https://github.com/InternLM/tutorial/assets/77949815/91fe5109-8068-461d-83bd-13d1fbf6e649)
这个问题的解决可以查看下Q&A文档哈~就是运行前引入下镜像就好了

**Comment by zengya55** [comment]
https://github.com/zengya55/InterLM_Learning/blob/main/lesson2/%E8%B6%A3%E5%91%B3DEMO%E7%AC%94%E8%AE%B0.md

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by jimmyrich7203** [comment]
第二节笔记 1609 
https://github.com/jimmyrich7203/InternLMlecture/blob/main/%E6%B5%A6%E8%AF%AD%20%E7%AC%AC%E4%BA%8C%E8%8A%82%20%E7%AC%94%E8%AE%B0.pdf

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by juxingyiwan** [comment]
https://github.com/juxingyiwan/InternLM_Camp_202401/blob/main/Lecture2/Lecture2%E8%AF%BE%E7%A8%8B%E7%AC%94%E8%AE%B0.pdf

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by SpongeBob-0715** [comment]
https://p065drvsp5m.feishu.cn/docx/FvUadw5ixoJNqtx8J9Ac8dNSnrR?from=from_copylink

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by FableFatale** [comment]
https://www.cnblogs.com/yunyetui/p/17957523

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by qqqmmmxxx** [comment]
https://zqm6a0l4f0l.feishu.cn/wiki/RRXrwffxGiSbIDkKtN8cvjAHnNh?from=from_copylink

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by MQDCY** [comment]
https://blog.csdn.net/weixin_48396757/article/details/135544808?spm=1001.2014.3001.5502

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by ykuang** [comment]
https://github.com/ykuang/internlm-tutorial/blob/main/notes/lesson2.md

**Reply by zcg-joker** [reply]
很棒哒~ 问题的记录很有用~

**Comment by jhyz** [comment]
https://blog.csdn.net/jinghangyz/article/details/135558418?spm=1001.2014.3001.5502

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by fxlt** [comment]
https://github.com/fxlt/internLM-Study/blob/main/2.%E8%BD%BB%E6%9D%BE%E7%8E%A9%E8%BD%AC%E4%B9%A6%E7%94%9F%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8Bdemo.md

**Reply by zcg-joker** [reply]
笔记尽量不要完全复制教程哈~

**Comment by 12345567890hc** [comment]
https://www.mubu.com/doc/3qvc2Vt0oSS

**Reply by zcg-joker** [reply]
很棒哒~
![image](https://github.com/InternLM/tutorial/assets/77949815/f4915f60-cb65-4b6d-89ac-4541233b89e9)
这个问题可以看看是不是ssh没有连接好哈~ 有问题可以在群里问哈~

**Comment by JJbond-sheng** [comment]
https://blog.csdn.net/2301_80135859/article/details/135560400?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135560400%22%2C%22source%22%3A%222301_80135859%22%7D

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by Alphonsinaone** [comment]
第2节课笔记
https://github.com/Alphonsinaone/InternLM/blob/main/Study%20Notes/Lesson_2.md

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by LordHidy** [comment]
https://github.com/LordHidy/InternLM_learning/blob/main/Notes/Lesson_2.md

**Reply by zcg-joker** [reply]
笔记尽量不要完全复制教程的文字哈~

**Comment by wade361** [comment]
https://github.com/wade361/-Wade/issues/2

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by liliang1918** [comment]
https://blog.csdn.net/lalala12ll/article/details/135560310?spm=1001.2014.3001.5501

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by IdealDestructor** [comment]
https://blog.deqiang.wang/2024/01/12/Project-LLM2/

**Reply by zcg-joker** [reply]
感觉是作业哈~

**Comment by GuoYiFantastic** [comment]
https://github.com/GuoYiFantastic/InternLM_training_camp/tree/main/%E8%BD%BB%E6%9D%BE%E7%8E%A9%E8%BD%AC%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%B6%A3%E5%91%B3Demo

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by wunuooo** [comment]
https://blog.csdn.net/ynx005/article/details/135577621?spm=1001.2014.3001.5502

**Reply by zcg-joker** [reply]
很棒哒~

**Comment by CharlineSu** [comment]
https://cewmxvz60v.feishu.cn/wiki/NaikwQCYuitJANkfp6CcZaVkn5c
